src/components/deck/index.js

```diff
@@ -273,7 +273,7 @@ Deck.propTypes = {
 };
 
 const ConnectedDeck = props => (
-  <ThemeProvider theme={theme}>
+  <ThemeProvider theme={mergeTheme(props.theme)}>
     <TransitionPipeProvider>
       <Deck {...props} />
```

Comment:
One thing I'm not sure on -- do I need to pass the merged `theme` here too? So like the equivalent of:

```js
<Deck {...props, ...{ theme: mergeTheme(props.theme) }} />
```

(But I'd do only one call to `mergeTheme()` in actual code).